android/src/main/java/com/github/rmtmckenzie/qrmobilevision/QrCameraC2.java

```diff
@@ -212,8 +211,8 @@ private Integer afMode(CameraCharacteristics cameraCharacteristics) {
         }
 
         @Override
-        public FirebaseVisionImage toImage() {
-            return FirebaseVisionImage.fromMediaImage(image, firebaseOrientation);
+        public InputImage toImage() {
+            return InputImage.fromMediaImage(image, firebaseOrientation);

```

Comment:
It appears that `InputImage` only accepts 0, 90, 180, 270 for the second parameter (rotationDegrees).
`getFirebaseOrientation` returns 0, 1, 2, 3.